testsuite-real/run-all

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+if [ "$(id -u)" != "0" ]; then
+   echo "These tests only work when run as root." >&2
+   echo "Yes. See also the CAUTION file." >&2
+   exit 1
+fi
```

Comment:
If I am not wrong, after installing the testsuite subpackage, there will be two scripts available to the user: *run-all* and *setup-and-run-all*. I think *run-all* is very dangerous, and we should avoid to expose it directly. What do you think about integrate *run-all* into *setup-and-run-all*?

Basicall, the function *run()* can be moved to *setup-and-run-all* and the rest of the script can be added to *main()* function.

Reply:
I think that would do little help, since the package also contains 16 other programs that do the individual tests.